hack/secrets.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+kerberos_id=$1
+kubeconfig=$2
+
+make kerberos_id=${kerberos_id} ci-secret-generator
```

Comment:
Would make sense to set `dry-run=false` here (or abort the whole script in the beginning if that is unset)